testCase.spec.ts

```diff
@@ -0,0 +1,14 @@
+export type ResultObject = {
+    fileName: string;
+    decompressFile?: boolean;
+};
+
+export type TestCase = {
```

Comment:
Maybe we could use zod to generate types? Then we could even validate at runtime whether the test case object is of the correct type.